data/net/minecraft/client/gui/font/glyphs/BakedGlyph.mapping

```diff
@@ -4,9 +4,13 @@ CLASS net/minecraft/client/gui/font/glyphs/BakedGlyph
 		ARG 2 seeThroughType
 		ARG 3 polygonOffsetType
 		ARG 4 u0
+			COMMENT minX position of the glyph texture
 		ARG 5 u1
+			COMMENT maxX position of the glyph texture
 		ARG 6 v0
+			COMMENT minY position of the glyph texture
 		ARG 7 v1
+			COMMENT maxY position of the glyph texture
```

Comment:
Seems strange to use `minX`, `maxX`, etc if they don't refer to variable names. I'd suggest rewording all of these to `min/max x/y coordinate` (`of the glyph texture` is also redundant).